plugins/framework16-keyboard/fu-framework16-keyboard.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2025 Framework Computer Inc
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+enum FuFramework16KeyboardResetRequest {
+    BootloaderJump      = 0x0B,
+}
+
+#[derive(New, Default)]
+#[repr(C, packed)]
+struct FuStructFramework16KeyboardResetRequest {
+    report_id: u8 == 0x00,
+    cmd: u8 == 0x0B,
+    reserved: [u8; 30], // == [0xFE; 30]
```

Comment:
@hughsie can I initialize a vector here?

Reply:
What would be the rust syntax to do that?

Reply:
https://github.com/fwupd/fwupd/pull/9110

I managed to do it :D